base/applications/mspaint/common.h

```diff
@@ -11,4 +11,6 @@
 
 /* FUNCTIONS ********************************************************/
 
-extern BOOL zoomTo(int, int, int);
+BOOL zoomTo(int, int, int);
+int Zoomed(int xy);
+int UnZoomed(int xy);
```

Comment:
I think that these two functions could be implemented in this file and marked as inlined.

Reply:
Needs a global variable and its class definition.

Reply:
If common.h gets included after inclusion of toolsModel in the source files that use it, then it should work (without the need for common.h itself to include that other .h) 🤔

Reply:
Fixed.